playout/install/systemd/libretime-liquidsoap.service

```diff
@@ -2,6 +2,8 @@
 Description=Libretime Liquidsoap Service
 
 [Service]
+Environment=LIBRETIME_LOG_FILEPATH=/var/log/libretime/liquidsoap.log
```

Comment:
This is part of system integration config, and I wonder if using a [EnvironmentFile](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=) instead of directly loading Environment inside the service file would make system integration easier. But this would mean a set of files to dispatch on the system (/etc/default/libretime-* or /etc/libretime/env).

For now we have 3 variables, I don't know if this will grow. And most of them should be similar (eg LIBRETIME_CONFIG_FILEPATH).

For now I am happy with this, but we can keep this in mind if it get's out of hand.